include/tvm/runtime/profiling.h

```diff
@@ -150,6 +151,26 @@ class Timer : public ObjectRef {
 Timer DefaultTimer(Device dev);
 
 namespace profiling {
+/*! \brief Wrapper for `Device` because `Device` is not passable across the
```

Comment:
Device cannot be put as part of object container. How about DeviceObject?

Reply:
DeviceObjectNode sounds a little weird. I think wrapper clearly explains what this is doing.

Reply:
I am not strongly attached to the choice, given it is in the profiling namespace, i will let you decide then